src/views/Viewer.vue

```diff
@@ -41,9 +41,9 @@
 		<template #actions>
 			<ActionButton
 				v-if="OCA.Sharing"
-				icon="icon-share-white-forced"
+				icon="icon-menu-sidebar-white-forced"
 				@click="showSharingSidebar">
-				{{ t('viewer', 'Share') }}
+				{{ t('viewer', 'Open sidebar') }}
```

Comment:
This is not only opening the sidebar, this is opening the sharing section of the sidebar.
Therefore if sharing is disabled, this button will be hidden. The showSharingSidebar needs to be updated accordingly alongside the `v-if="OCA.Sharing"` :)

Reply:
For all intents and purposes, this button is the only one in Viewer which opens the sidebar. :) So that’s what it should be called.

> Therefore if sharing is disabled, this button will be hidden.

And then there’s also no way to open the sidebar from the viewer, right?

Therefore I think we should merge this and move any further updates to the menu implementation PR → https://github.com/nextcloud/viewer/pull/15

Reply:
No, that would leave the current state of the viewer in a not optimal one. :)
Please take the time to adjust this pr instead!

#15 is much more work

Reply:
ahh ok. of course it should also be possible to open the sidebar even if sharing is disabled.

Reply:
> No, that would leave the current state of the viewer in a not optimal one. :)

I don’t understand how, because:
- If Sharing is enabled, the button is shown and it _does_ open the sidebar.
- If Sharing is _not_ enabled, the icon/button is not shown.

So what about that is not optimal, or less optimal than now?

If it should be changed to not rely on Sharing I would love to do that but need to know how. :)

Reply:
Ah, yes, you're right.
My point was that now that we tell the user that the button open the sidebar, s.he might get confused when the button disappear if sharing is disabled. Because there is still other stuff you can do in the sidebar :)